### PR TITLE
Feature/plans

### PR DESCRIPTION
**Pull request by mansona**
Implementing CRUD for Accounts, Plans and Permissions

### CODE REVIEW

File: app/accounts/edit/template.hbs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{{#paper-dialog title="Edit Account" close="cancel" classNames="edit-dialog"}}
+  <md-dialog-content>
```
Comment (ynnoj) [inline app/accounts/edit/template.hbs:2]: @mansona What is this crazy tag?

File: app/styles/_circles.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  'X': 'indigo',
+  'Y': 'blue',
+  'Z': 'light-blue'
+);
+
+@each $letter, $color in $alphabet {
+  .circle--#{$letter}{
+    background: color($color, '200');
+  }
+}
```
Comment (ynnoj) [inline app/styles/_circles.scss:51]: So proud :')

File: app/user/model.js [hunk 3]
```diff
@@ -13,5 +13,26 @@ export default DS.Model.extend({
...
+      }).then(() => {
+        return resolve(true);
+      });
+    });
+
+    return DS.PromiseObject.create({
+      promise: promise
+    });
+
+  }),
```
Comment (thedoktor) [inline app/user/model.js]: remove comma

**Comment by mansona** [comment]
thanks! merging :+1:

**Pull request merged**